journals/apps/journals/templates/videos/import_video.html

```diff
@@ -0,0 +1,146 @@
+
+    <div class="nice-padding" id="import-videos-container">
+        <div class="drop-zone">
+            <form action="{% url 'journals:import_videos' %}" method="POST">
+                {% csrf_token %}
+                <ul class="fields">
+                    <li class="message">{% trans "Import Videos" %}</li>
+                    {% if journals %}
+                        <li>
+                            <label for="id_import_video_journal">{% trans "Journal:" %}</label>
+                            <div class="field-content">
+                                <select id="id_import_video_journal" name="journal">
+                                    {% for journal in journals %}
+                                        <option value="{{ journal.id }}">{{ journal.name }}</option>
+                                    {% endfor %}
+                                </select>
+                                <p class="help">{% trans "Select a journal to import videos into it" %}</p>
+                            </div>
+                        </li>
+                    {% endif %}
+                    <li>
+                        <label>{% trans "Course runs:" %}</label>
+                        <div class="field-content">
+                            <ul class="field-content course-runs">
+                                <li>
+                                    <input type="checkbox" name="course_runs_all" value="" id="course_run_all">
+                                    <label for="course_run_all">All</label>
+                                    <span class="help">{% trans "Select course runs to find videos from" %}</span>
+                                </li>
+                            </ul>
+                        </div>
+                    </li>
+                    <li>
+                        <label for="id_import_video_collection">{% trans "Collection:" %}</label>
+                        <div class="field-content">
+                            <select id="id_import_video_collection" name="collection">
+                                {% for collection in collections %}
+                                    <option value="{{ collection.id }}">{{ collection.name }}</option>
+                                {% endfor %}
+                            </select>
+                            <p class="help">{% trans "Select a video collection which would be associated to imported videos" %}</p>
+                        </div>
+                    </li>
+                    <li>
+                        <span class="working icon"></span><button type="submit" class="button bicolor icon icon-plus">{% trans "Import Videos" %}</button>
+                    </li>
+                </ul>
+            </form>
+        </div>
+        <div><hr></div>
+        <div class="drop-zone">
+            <div class="import-status">{% trans "Import Status:" %}&nbsp;<span></span></div>
```

Comment:
Regarding Import Status text messages, please make the following changes:
- Change `Import Status: Completed` to `Import Status: Completed, {num_videos) videos imported`
- Left justify the text "Imported title from course" so they all left align, currently all align center
- Change the message from `Imported title from course` to `Imported 'title' from 'course'`